File: benchmark/results/java-invocation-script/run-004/b-baseline/output/com/metaengine/demo/identity/services/UserRepository.java
package com.metaengine.demo.identity.services;

import com.metaengine.demo.identity.aggregates.User;
import java.util.List;

/** UserRepository persistence service in the identity domain. */
public class UserRepository {
    /** Persist a new User from a partial input. */
    public User create(User input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a User by id, or null if not found. */
    public User findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Users up to limit. */
    public List<User> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a User by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-invocation-script/run-003/a-mcp/output/identity/services/user_repository.py
from identity.aggregates.user import User

"""UserRepository service."""
class UserRepository:
    def create(self, input: User) -> User:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> User | None:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[User]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

